subcommittees/mentoring/minutes-2015-06-26.md

```diff
@@ -0,0 +1,122 @@
+# Software Carpentry Mentoring Subcommittee Meeting
+
+Date: 26/06/2015
+
+Time: 5pm GMT
+
+## Attending
+
+-   Karin Langesen
+-   Kate Hertweck
+-   Raniere Silva
+
+## Apologies
+
+-   Carol Willing
+-   Daniel Chen
+-   Sheldon McKay
+-   Tiffany Timbers
+
+## Actions
+
+-   RS: check with Greg the schedule for Instructor Training.
+
+## Announcements
+
+-   We are hiring a program coordinator: http://software-carpentry.org/blog/2015/06/program-coordinator-position.html
+-   Daniel Chen wrote a blog post about assessing: http://software-carpentry.org/blog/2015/06/learner-assessment-part-01.html
+-   Greg Wilson wrote a blog post about the instructor training: http://software-carpentry.org/blog/2015/06/training-lessons.html
+
+## Past Actions
+
+-   RS send email to Daniel
+
+    Done.
+-   ALL: Review https://github.com/swcarpentry/board/pull/43
+-   RS: Archieve last minute meetings
+
+    PR created: https://github.com/swcarpentry/board/pull/41
+
+    Pull request merged.
+-   TT: Road map for micro lesson
+
+    Pull reques created: https://github.com/swcarpentry/board/pull/43
+
+## Debriefing Sessions
+
+-   June 23
+
+    Thanks to Kate and Sheldon.
+
+    Post at http://software-carpentry.org/blog/2015/06/instructor-debriefing-2015-06-24.html
+
+    KH: Some things are always mention on debriefing sessions.
+
+    KL: I will suggest that the new Git lesson that everyone works on their on planet but when moving to GitHub people just work on their pair.
+
+    KH: This is better than just append the name?
+
+    KL: Git isn't the more stable lesson in our curriculum.
+
+    KH: People are teaching branch. Maybe add it to the lesson? Maybe create a issue when things are common on debriefing.
+
+    KL: Maybe keep a spreedsheet. Can you start this and give access to lesson maintainers?
+
+    KH: Instructors complain about Vlad and Wolfman.
+
+    KL: Ridiculous but works. I used it as a ice break for who has the red as favorite color as red.
+
+    KL: What I will like to have is "This challenge hasn't solution."
+
```

Comment:
As in - many challenges do not have solutions. I would like to have solutions for all of them.